### ISSUE TITLE
Issue persists on MC1.20.1 AE-Additions 5.0.5 ATM9

### ISSUE THREAD

**Issue by AnthonyOGorman** [the9grounds/AE-Additions#153]
Issue persists on MC1.20.1 AE-Additions 5.0.5 ATM9 (Running Server Side)
![image](https://github.com/the9grounds/AE-Additions/assets/1390977/410218ce-eb3c-46f5-9460-b061ffe7c704)
Unable to create new channels, however even though no channels are showing, existing transceivers are still connected.

_Originally posted by @AnthonyOGorman in https://github.com/the9grounds/AE-Additions/issues/133#issuecomment-2013094717_

I did some digging as this issue seems to be caused by the FTB-Teams mod which has a "API break" on version [2001.2.0](https://github.com/FTBTeam/FTB-Teams/releases/tag/v2001.2.0)
Specifically `FTBTeamsAPI.arePlayersInSameTeam()` changes to `FTBTeamsAPI.api().getManager().arePlayersInSameTeam()`

TLDR;
The bug depends on what version of FTB-Teams your modpack decided to run with.
You can fix the issue by commenting out the FTBTeams parts in ChannelInfo.kt or updating the references.

**Comment by RadegasTCZ** [comment]
Is it just me or I am unable to find that file to 'Fix' this

**Comment by AnthonyOGorman** [comment]
The "ChannelInfo.kt" file is located [here](https://github.com/the9grounds/AE-Additions/blob/master/src/main/kotlin/com/the9grounds/aeadditions/util/ChannelInfo.kt).
Or if you would rather just update the dependency checkout the above [PR](https://github.com/the9grounds/AE-Additions/pull/154).
